lib/src/directus_api_manager_base.dart

```diff
@@ -389,17 +392,21 @@ class DirectusApiManager implements IDirectusApiManager {
 
   @override
   Future<Type> updateItem<Type extends DirectusData>(
-      {required Type objectToUpdate, String? fields}) {
+      {required Type objectToUpdate,
+      String? fields,
+      bool forceSaving = false}) {
```

Comment:
I think I'd prefer the name `force` which is more commonly used. 
Also could you add some doc comments to the function and its parameters ? since you deprecated the `updateDurectusUser` rerouting it here, it would help everyone how this function should be used.

Reply:
This is done in e7fc92b51b105d5831d5f0e99335feaa407e58ea